buildconfig/CMake/LinuxPackageScripts.cmake

```diff
@@ -46,19 +45,28 @@ file ( WRITE ${CMAKE_CURRENT_BINARY_DIR}/mantid.csh
   "setenv MANTIDPATH \"${CMAKE_INSTALL_PREFIX}/${BIN_DIR}\"\n"
   "setenv PV_PLUGIN_PATH \"${CMAKE_INSTALL_PREFIX}/${PVPLUGINS_DIR}/${PVPLUGINS_DIR}\"\n"
   "setenv PATH \"\${PATH}:\${MANTIDPATH}\"\n"
-
-  "if ($?PYTHONPATH) then\n"
-  "  setenv PYTHONPATH \"\${MANTIDPATH}:\${PYTHONPATH}\"\n"
-  "else\n"
-  "  setenv PYTHONPATH \"\${MANTIDPATH}\"\n"
-  "endif\n"
 )
 
 install ( PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/mantid.sh
   ${CMAKE_CURRENT_BINARY_DIR}/mantid.csh
+  ${CMAKE_CURRENT_BINARY_DIR}/mantid.pth
   DESTINATION ${ETC_DIR}
 )
 
+###########################################################################
+# Find python site-packages dir and create mantid.pth
+###########################################################################
+execute_process(
+  COMMAND "${PYTHON_EXECUTABLE}" -c "if True:
```

Comment:
Just out of curiosity, why is the `if True:` needed?

Reply:
It's not. I wrote this 6 months ago and I can't remember why I had that, probably from where I copied it. 😆

Reply:
:)